### DISCUSSION TITLE
How can I implement social authentication (e.g., login with Google or Facebook) in a Next.js application?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#106668] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

Social Authentication

**Comment by gil-son** [comment]
Hello,

look that [video](https://www.youtube.com/watch?v=AbUVY16P4Ys)

Good Luck!

**Comment by SRayen** [comment]
Try to use NextAuth,
NextAuth.js Documentation: https://next-auth.js.org/

**Comment by amankr1469** [comment]
With the help of NextAuth.js and credentials which you get from Firebase you can use social authentications.
